Tests/SwiftParserTest/MemberRefTests.swift

```diff
@@ -0,0 +1,233 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import SwiftParser
+import XCTest
+
+final class MemberTypeTests: XCTestCase {
+  func testKeyword() {
+    AssertParse(
+      "MyType.class",
+      { TypeSyntax.parse(from: &$0) },
+      substructure: Syntax(MemberTypeIdentifierSyntax(
+        baseType: SimpleTypeIdentifierSyntax(
+          name: .identifier("MyType")
+        ),
+        period: .period,
+        name: .identifier("class")
+      )))
+  }
+
+  func testMissing() {
+    AssertParse(
+      "MyType.1️⃣",
+      { TypeSyntax.parse(from: &$0) },
+      substructure: Syntax(MemberTypeIdentifierSyntax(
+        baseType: SimpleTypeIdentifierSyntax(
+          name: .identifier("MyType")
+        ),
+        period: .periodToken(),
+        name: .identifier("", presence: .missing)
+      )),
+      diagnostics: [DiagnosticSpec(message: "expected name in member type")])
+  }
+
+  func testValidWhitespace() {
+    let expected = Syntax(MemberTypeIdentifierSyntax(
+      baseType: SimpleTypeIdentifierSyntax(
+        name: .identifier("MyType")
+      ),
+      period: .periodToken(),
+      name: .identifier("Nested")
+    ))
+
+    let cases = [
+      "MyType .Nested",
+      "MyType/*foo*/.Nested",
+      "MyType\n.Nested",
+      "MyType . Nested",
+      "MyType/*foo*/./*bar*/Nested",
+      "MyType\n.  Nested",
+      "MyType\n.\nNested"
+    ]
+    for source in cases {
+      AssertParse(
+        source,
+        { TypeSyntax.parse(from: &$0) },
+        substructure: expected)
+    }
```

Comment:
What do you think about adding `#line` as a key to `cases` like in the following? That way Xcode will point you to the test case that failed instead of the `AssertParse` call that is being repeated.

https://github.com/apple/swift-syntax/blob/fd2f742c7bdac60f27685ebdacce146706f1d8ef/Tests/SwiftSyntaxBuilderTest/TriviaTests.swift#L49-L64

Reply:
Sure